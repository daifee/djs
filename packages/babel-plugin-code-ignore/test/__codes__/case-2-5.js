



function fun() {

  console.log('error');// @code-ignore-statement kuwo
  console.log('success'); // @code-ignore-statement kugou
  console.log('error1'); // @code-ignore-statement kuwo
}