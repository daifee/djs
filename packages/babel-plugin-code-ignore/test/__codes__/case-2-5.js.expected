"use strict";

function fun() {
  //
  console.log('success'); // @code-ignore-statement kugou
  //
}